### PR TITLE
[SITE-2833] PHP 8.1+ patch releases

### PR DESCRIPTION
**Pull request by pwtyler**
## Summary

**[Release Notes](https://docs.pantheon.io/release-notes)** - Adds release note for recent PHP patch releases.

### CODE REVIEW

File: source/releasenotes/2024-11-22-php-81-82-83.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "PHP 8.1, 8.2 and 8.3 updated to their latest patch releases"
```
Comment (pwtyler) [inline source/releasenotes/2024-11-22-php-81-82-83.md:2]:
```suggestion
title: "PHP 8.1, 8.2 and 8.3 updated to their latest security releases"
```

Maybe worth calling out that this is a security release?

Comment (scottbuscemi) [reply source/releasenotes/2024-11-22-php-81-82-83.md:2]: The 8.3/8.2 releases also include bug fixes. I think patch is fine.

File: source/releasenotes/2024-11-22-php-81-82-83.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+published_date: "2024-11-22"
+categories: [infrastructure, security]
+---
+PHP versions [8.1.31](https://www.php.net/ChangeLog-8.php#8.1.31), [8.2.26](https://www.php.net/ChangeLog-8.php#8.2.26) and [8.3.14](https://www.php.net/ChangeLog-8.php#8.3.14) are now available on the platform, bringing the latest bug fixes and updates for improved performance and security.
```
Comment (scottbuscemi) [inline source/releasenotes/2024-11-22-php-81-82-83.md:6]:
```suggestion
PHP versions [8.1.31](https://www.php.net/ChangeLog-8.php#8.1.31), [8.2.26](https://www.php.net/ChangeLog-8.php#8.2.26) and [8.3.14](https://www.php.net/ChangeLog-8.php#8.3.14) are now available on the platform, bringing the latest bug fixes and enhancements for improved performance and security. Sites will automatically receive the updates in the coming days - no manual action is needed.
```

**Review by scottbuscemi — APPROVED** [review]

**Comment by IngridKwok** [comment]
Made some minor changes in the opening paragraph. Good to go.

**Pull request merged**